src/storage/store/node_local_state.rs

```diff
@@ -29,7 +29,29 @@ pub mod onchain_events {
 
     pub fn get_state(db: &RocksDB) -> Result<Option<OnChainEventState>, super::IngestStateError> {
         match db.get(&make_primary_key())? {
-            Some(hub_state) => Ok(Some(OnChainEventState::decode(hub_state.as_slice())?)),
+            Some(state) => Ok(Some(OnChainEventState::decode(state.as_slice())?)),
+            None => Ok(None),
+        }
+    }
+}
+
+pub mod fname_transfers {
```

Comment:
Let's make this a struct, and make the functions less generic (set_latest_transfer_id/get_latest_transfer_id)